packages/components/src/Lettera/Models.purs

```diff
@@ -302,18 +305,46 @@ data CategoryType
   | Webview
   | Link
 
+toString :: CategoryType -> String
+toString Feed = "feed"
+toString Webview = "webview"
+toString Link = "link"
+
 instance categoryTypeDecodeJson :: DecodeJson CategoryType where
   decodeJson json = do
     categoryTypeString <- decodeJson json
-    case String.toLower categoryTypeString of
+    case toLower categoryTypeString of
       "feed"    -> Right Feed
       "webview" -> Right Webview
       "link"    -> Right Link
       _         -> Left $ UnexpectedValue json
 
+newtype CategoryLabel = CategoryLabel String
```

Comment:
Tags have a similar representation issue going on with them, they also need a "display" and a "URL" style of them.

Reply:
Right. Maybe we could have a more general type for these.